frontends/web/src/style/icons.css

```diff
@@ -0,0 +1,41 @@
+.checked-circle {
+    stroke-dasharray: 166;
+    stroke-dashoffset: 166;
+    stroke-width: 1;
+    stroke-miterlimit: 10;
+    stroke: #7ac142;
```

Comment:
TODO: check the green color, should we change the existing green https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/frontends/web/src/style/variables.css#L4

cc @jadzeidan

Reply:
fixed in https://github.com/digitalbitbox/bitbox-wallet-app/compare/c35550a2a18fb97e23c80966fd047a126bb3068c..16f6593e1dd6920e88a12c47c7ee539299fdbe88